councilmatic_core/management/commands/import_data.py

```diff
@@ -964,10 +964,19 @@ def insert_raw_bills(self, delete=False):
             with open(os.path.join(self.bills_folder, bill_json)) as f:
                 bill_info = json.loads(f.read())
 
+
+            # Add the API source, if available, which allows for easier debugging 
+            # Otherwise, assume that the 'web' source is available
             source_url = None
+            source_note = None
             for source in bill_info['sources']:
-                if source['note'] == 'web':
+                if source['note'] == 'api':
```

Comment:
if i'm reading this correctly, if there is *both* an api source and a web source, this code will prefer the api source. I think we want the opposite behavior, don't we?

Reply:
Oh, I was thinking we'd want the API source, but actually we should return the web url, since that gets rendered in some of Councilmatic display logic. Okee.